bio/emu/collapse-taxonomy/wrapper.py

```diff
@@ -0,0 +1,37 @@
+__author__ = "Curro Campuzano Jimenez"
+__copyright__ = "Copyright 2024, Curro Campuzano Jimenez"
+__email__ = "campuzanocurro@gmail.com"
+__license__ = "MIT"
+
+
+from snakemake.shell import shell
+import tempfile
+import os
+
+log = snakemake.log_fmt_shell(stdout=True, stderr=True)
+
+input_file = snakemake.input[0]
+if not isinstance(input_file, str) and len(snakemake.input) != 1:
+    raise ValueError(
+        "Input should be one TSV file generated with emu: " + str(input_file) + "!"
+    )
+
+output_file = snakemake.output[0]
+if not isinstance(output_file, str) and len(snakemake.output) != 1:
+    raise ValueError("Output should be one file: " + str(output_file) + "!")
+
+if not snakemake.params.get("rank"):
+    raise ValueError("Please provide a rank parameter")
+rank = snakemake.params.get("rank")
```

Comment:
Maybe just set up a default?
```suggestion
rank = snakemake.params.get("rank", "species")
```